test/oats/kafka/yaml/oats_java_kafka.yaml

```diff
@@ -10,28 +10,26 @@ expected:
   traces:
     - traceql: '{ .messaging.operation.type = "process" }'
       spans:
-        - name: 'my-topic process'
+        - name: 'regex:process$'
           attributes:
-            messaging.destination.name: my-topic
             messaging.operation.type: process
             messaging.system: kafka
-            #server.port: "9092"
+            server.port: "9092"
     - traceql: '{ .messaging.operation.type = "publish" }'
       spans:
-        - name: 'my-topic publish'
+        - name: 'regex:publish$'
           attributes:
-            messaging.destination.name: my-topic
             messaging.operation.type: publish
             messaging.system: kafka
-            #server.port: "9092"
+            server.port: "9092"
   metrics:
-    - promql: 'messaging_publish_duration_count{messaging_system="kafka", messaging_destination_name="my-topic"}'
```

Comment:
Is this removed on purpouse?

Reply:
Yes, we don't always find the topic in Java. I've no idea why, but we need to fix the flaky tests.

Reply:
that's possible depend on the protocol, but shouldn't be `messaging_destination_name="*"` now?